lib/ansible/modules/network/nxos/nxos_hsrp.py

```diff
@@ -165,14 +143,10 @@ def execute_show_command(command, module, command_type='cli_show'):
 
 def apply_key_map(key_map, table):
     new_dict = {}
-    for key, value in table.items():
+    for key in table:
         new_key = key_map.get(key)
         if new_key:
-            value = table.get(key)
-            if value:
-                new_dict[new_key] = str(value)
-            else:
-                new_dict[new_key] = value
+            new_dict[new_key] = str(table.get(key))
```

Comment:
This might be an instance where we don't actually want this, I doubt `'None'` is a proper value here